### DISCUSSION TITLE
Timeline for final 9.0.0 version

### DISCUSSION THREAD

**Discussion by OzzyTheGiant** [directus/directus#6771] [Archive]
Just curious, how much longer is there left to finish up 9.0? When I started using this platform, I entered at rc-57, then realized that I needed email integration and thankfully, rc-62 was around when I need this feature, but in the process, a few things broke, which was something that was warned about but didn't realize at the moment. Given that I've been working on other things and haven't been able to upgrade a current project to the latest rc, I feel that I'm doing fairly ok at rc-62 and would prefer to upgrade Directus only once 9.0 is finalized so that I can deal with breaking changes all in one go, at my own time. So yeah, that's the context of this question.

**Comment by benhaynes** [comment]
Can't say with certainty, but 9.0.0 will probably be released within the 4 weeks. We've been first trying to get a few last fixes/features in (specifically ones that may incur schema/breaking changes), and also get some new engineers hired on the team. Once released, we want to be ready to handle more daily activity... so we're trying to get ahead of things.

We realize this is annoying for users, and are trying to get this figured out asap.

**Reply by hermesalvesbr** [reply]
Thanks Ben for your compromise with the fans of directus, like me.

**Comment by OzzyTheGiant** [comment]
@benhaynes Noticed a few weeks ago that 9.0.0 finally hit. I think I might have some time to upgrade a website that I was putting off for a while now. Currently, I'm still down at rc-62. Would you say I should upgrade incrementally, or can I just jump all the way to the current version (9.4.x)? I can fix all breaking changes in one go but not sure if the version differences are too big for me.

**Reply by rijkvanzanten** [reply]
You should be able to upgrade to the latest version in one go (do make sure to backup everything of course). All migrations are ran in order, so doing them one by one vs all at the same time should end in the same result 👍🏻
